invenio_users_resources/services/users/service.py

```diff
@@ -34,6 +42,74 @@ def user_cls(self):
         """Alias for record_cls."""
         return self.record_cls
 
+    @unit_of_work()
+    def create_via_admin(self, identity, data, raise_errors=True, uow=None):
+        """Create a user."""
+        self.require_permission(identity, "create")
+        # Remove the following from data dict as will fail schema validation.
+        for key in [
+            "id",
+            "domain",
+            "preferences",
+            "profile",
+            "identities",
+            "domaininfo",
+        ]:
+            if key in data:
+                data.pop(key)
+        data, errors = self.schema.load(
+            data,
+            context={"identity": identity},
+            raise_errors=raise_errors,
+        )
+        # create flask user
+        user, errors = self._create_user_as_admin(data)
+        # run components
+        self.run_components(
+            "create",
+            identity,
+            data=data,
+            user=user,
+            errors=errors,
+            uow=uow,
+        )
+        # get email token and reset info
+        account_user = current_datastore.get_user(user.id)
+        token, reset_link = default_reset_password_link_func(account_user)
+        # trigger celery task to send email.
+        uow.register(
```

Comment:
I wonder why they send the email inline (apparently) on the "regular" user endpoints https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/views/rest.py#L396